students/swietczak_monika/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,7 @@
+# Read the numbers like this:
+n = int(input())
+k = int(input())
+# Print the result with print()
+
+print(k//n)
+print(k%n)
```

Comment:
Please remove all PEP8 warning in your PR.

Warning is underlined with the orange curvy line. If you hover your mouse pointer over this curvy line, the detailed information about the error appears at the tooltip.
![image](https://user-images.githubusercontent.com/8876350/36839685-07d9f5a2-1d43-11e8-9cee-a8ceefe318b6.png)


Even better soultion, always check you code before PR is created with inspect tool. Just select 'Code' -> 'Inspect Code'  in PyCharm IDE
![image](https://user-images.githubusercontent.com/8876350/36839530-aacd3fae-1d42-11e8-8c35-5580f29b80f4.png)